master/networking/migrate-pools.md

```diff
@@ -0,0 +1,198 @@
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+- **IPPool** resource 
+
+### Concepts
+
+#### IP pools and cluster CIDRs
+
+{{site.prodname}} supports changing IP pools for pods if the IP pool you are changing to is within the same cluster CIDR. Although it is technically feasible to change a pod to an IP pool outside the cluster CIDR, we do not recommend it; this requires restarting components (affecting application availability), and if you change to a CIDR range that doesn’t include the already-assigned value, existing pods will lose connectivity.
+
+### Before you begin...
+
+- You must be using {{site.prodname}} IPAM. 
+
+  To verify, ssh to one of your Kubernetes nodes and view the CNI configuration.  
+
+  `cat /etc/cni/net.d/10-calico.conflist`
+
+  Look for the "type" entry:
+
+  <pre>
+     "ipam": {
+           "type": "calico-ipam"
+      }, 
+  </pre>
+
+   If the type is “calico-ipam,” you are good to go. If the IPAM is set to something else, or the 10-calico.conflist file does not exist, you cannot use this feature in your cluster. 
+
+- Your orchestrator must support changing the pod network CIDR.   
+  Although Kubernetes supports changing the pod network CIDR, not all orchestrators do. For example, OpenShift does not support this feature as described in [`osm_cluster_network_cidr configuration`](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables). Check your orchestrator documentation to verify. 
+
+### How to
+
+#### Migrate from one IP pool to another
+
+Follow these steps to migrate pods from one IP pool to another pool. 
+
+> **Important!** If you follow these steps, existing pod connectivity will not be affected. (If you delete the old IP pool before you create and verify the new pool, existing pods will be affected.) When pods are deleted, applications may be temporarily unavailable (depending on the type of application); plan accordingly. 
+{: .alert .alert-danger }
+
+1. Add a new IP pool.  
+   **Note**: The new IP pool must be within the same cluster CIDR.
+1. Disable the old IP pool.  
+   **Note**: Disabling an IP pool only prevents new IP address allocations; it does not affect the networking of existing pods.
+1. Delete pods from the old IP pool.  
+   This includes any new pods that may have been created with the old IP pool prior to disabling the pool.
+1. Verify that new pods get an address from the new IP pool.
+1. Delete the old IP pool.
+
+### Tutorial
+
+In the following example, we created a Kubernetes cluster using **kubeadm**. When we installed calico nodes in this cluster, a default IP pool was assigned: **192.168.0.0/16**. Now, we want pods in the cluster to use IPs in the CIDR: **10.0.0.0/16** (within the cluster CIDR). 
```

Comment:
"calico" -> {{site.prodname}}

We might want to call out that in the example we accidentally used the wrong CIDR and want to change it.

We should make it clear this isn't the default install path for kubeadm.

Reply:
Fixed.